README.md

```diff
@@ -1,3 +1,273 @@
-# Garden Setup
+# Install Gardener on your Kubernetes Landscape
 
-This repository describes the Gardener product comprising of components which need to be installed for a Gardener landscape. A Gardener landscape can be installed using [_Sow_](https://github.com/gardener/sow)  based on the Gardener product definition in _garden-setup_ and a landscape configuration parametrizing the actual landscape to be installed.
\ No newline at end of file
+Gardener uses Kubernetes to manage Kubernetes clusters. This documentation describes how to install Gardener on an existing Kubernetes cluster of your IaaS provider.
+> Where reference is made in this document to the *base cluster*, we are actually referring to the existing cluster where you will install Gardener. This helps to distinguish them from the clusters that you will create after the installation using Gardener. Once it's installed, it is also referred to as *garden cluster*. Whenever you create clusters, Gardener will create *seed clusters* and *shoot clusters*. In this documentation we will only cover the installation of clusters in one region of one IaaS provider. More information: [Architecture](https://gardener.cloud/030-architecture/).
+
+## Prerequisites
+
+* The installation was tested on Linux and MacOS
+* You need to have the following tools installed:
+  * [Docker](https://docs.docker.com/install/)
+  * [git](https://git-scm.com/downloads)
+  * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
```

Comment:
One also needs `make` and `yaml2json` if you want to include that.

Reply:
The sow image is now available in a registry, so `make` is only needed if you want to build it yourself. 
We should be able to get rid of the `yaml2json`, I think.

Reply:
With https://github.com/gardener/garden-setup/commit/d31f816f31e848331baab5daace5d92fef5d7203 the `yaml2json` dependency (and one to `jq`) should be gone.

Reply:
Isn't `make` still required to build to Docker image?